from unittest import TestCase


class TestInceptionMerge(TestCase):
    def test_forward(self):
        self.fail()
